static/app/views/profiling/landing/profilingSlowestTransactionsPanel.tsx

```diff
@@ -147,7 +147,7 @@ function SlowestTransactionPanelItem({
   return (
     <PanelItem key={transaction.transaction}>
       <Flex justify="space-between" gap={space(1)}>
-        <PlatformIcon platform={transactionProject?.platform} />
+        <PlatformIcon platform={transactionProject?.platform || 'default'} />
```

Comment:
```suggestion
        <PlatformIcon platform={transactionProject?.platform ?? 'default'} />
```

Is this not correctly reflected in the `PlatformIcon` types?

Reply:
Hm... It looks like types aren't being checked for `PlatformIcon`, any ideas why that is? I can pass anything as `platform` and nothing complains

Reply:
hmm, it should be type string only https://github.com/getsentry/platformicons/blob/master/src/platformIcon.tsx#L152

Reply:
yeah i see that, but locally, it's not complain no matter what i pass it. null/undefined/objects/number all pass the type checking